fiberzap/config.go

```diff
@@ -41,6 +42,11 @@ type Config struct {
 	//
 	// Optional. Default: {"Server error", "Client error", "Success"}
 	Messages []string
+
+	// Custom response levels.
+	//
+	// Optional. Default: {zapcore.ErrorLevel, zapcore.WarnLevel, zapcore.InfoLevel}
+	Levels []zapcore.Level
```

Comment:
please add something in the readme in the config section for this

Reply:
done